### ISSUE TITLE
Sorting Issue

### ISSUE THREAD

**Issue by manotiwa** [nicolaskruchten/pivottable#337]
Hi Nicolas, I want to short a column in descending order of a column value so can you suggest me the code , There is no place or button to short a column in ascending or descending order , I need this on urgent , please suggest me what can I do .

**Comment by nate-fr** [comment]
Export the resulting table in office calc or excel, then sort.

Seems your issue is getting smaller.

**Comment by nicolaskruchten** [comment]
Hello @manotiwa,

Unfortunately, table sorting is not a feature of this library: https://github.com/nicolaskruchten/pivottable/wiki/Frequently-Asked-Questions#sorting-in-built-in-table-renderer

**Comment by manotiwa** [comment]
Hi Sir,

Is there any other option that I can do to accomplish this task, please
help as Im new to this kind of feature.

Please help.

Regards,
Manoranjan Tiwari

**Comment by nicolaskruchten** [comment]
Unfortunately there's not much I can suggest to help you, other than adding in your own dynamic sorting to the table with something like http://tablesorter.com/

**Comment by cathgr** [comment]
Good day Nicolas,
I have created a pivot table in R using the rpivottable (which I found absolutely amazing!). However, as I am dealing with a lot of data and am interested in the largest values, I need to sort my values in descending order. On this last post in 2015, you proposed to use the third party tablesorter. But, with my lack of experience, I found it quite difficult to implement in my R code so far...  Since last year, I was wondering if you have any suggestion for less complexity for a beginner/intermediary R user ?
Thank you very much, 
Catherine
